test/package_test.js

```diff
@@ -0,0 +1,15 @@
+suite('Package', () => {
+  let assert = require('assert');
+  let pack = require('../package.json');
```

Comment:
Have you checked this, I don't think you can just require a json file...
You might need to do read file and JSON.parse... but maybe node has an exception for package.json..

I guess we'll see if it works :)